metadata/javax.cache/cache-api/1.1.1/serialization-config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "lambdaCapturingTypes": [],
+  "proxies": [],
+  "types": [
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "javax.cache.configuration.FactoryBuilder$SingletonFactory"
+    },
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "javax.cache.expiry.AccessedExpiryPolicy"
+    },
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "javax.cache.expiry.Duration"
+    },
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "javax.cache.expiry.EternalExpiryPolicy"
+    },
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "java.lang.String"
+    },
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "java.lang.Number"
+    },
+    {
+      "condition": {
+        "typeReachable": "java.lang.Object"
+      },
+      "name": "java.lang.Integer"
+    }
```

Comment:
- I was a bit surprised when I manually added these 3 GraalVM reachability metadata entries from nativeTest 's Error Log. This looks like something GraalVM needs to do, but now requires me to add them manually.